terraform/aptos-node/aws/outputs.tf

```diff
@@ -0,0 +1,56 @@
+output "helm_release_name" {
```

Comment:
most of the outputs here doesn't make sense to print out anymore, do some cleanup?

Reply:
They're actually quite useful, or used by the parent `aptos-node-testnet` root module.